src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSJobSchedulingError.cs

```diff
@@ -45,7 +45,7 @@ internal PSJobSchedulingError(Microsoft.Azure.Batch.JobSchedulingError omObject)
             this.omObject = omObject;
         }
         
-        public Microsoft.Azure.Batch.Common.SchedulingErrorCategory Category
+        public Microsoft.Azure.Batch.Common.ErrorCategory Category
```

Comment:
Does it count "breaking change"?

Reply:
I am not sure -- one of the Powershell team members can comment maybe?

Reply:
If it has a superset of the properties of the previous type, then it is not a breaking change.  Otherwise, it is a breakign change.

Reply:
Yes, it has a superset.